src/components/attendee/scss/Gradients.module.scss

```diff
@@ -1,5 +1,4 @@
 .Gradients {
```

Comment:
[Q] Is this file in the right place? Why is there a shared `scss` subdirectory of `... /components/attendee/ ...` ?